src/components/FuncCard.js

```diff
@@ -91,16 +96,18 @@ class FuncCard extends Component {
             onParamsChange={this.handleParamsChange}
             onSubmit={this.handleParamSubmit}
         />
-
-        <Highlight lang="js" value={callCode} />
-
+        <div className="results-div">
+          <Button floating className='grey darken-3 small-btn' waves='light' icon='clear' onClick={this.handleClearCode}>Clear</Button>
```

Comment:
Disabling the button when there are no params might be good.
```js
disabled={params.length === 0}
```

Reply:
I like that - and if there is a value in result